### PR TITLE
transition to home when click logo

### PR DESCRIPTION
**Pull request by Olga1984**

### CODE REVIEW

File: client/src/shared/header/header.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <header class="wk-header" role="banner">
     <swt-hamburger-menu></swt-hamburger-menu>
     <div class="wk-header-container">
-        <a class="wk-logo" href="javascript:void(0)">
+        <a class="wk-logo" ui-sref="{{header.state}}">
```
Comment (gshokanov) [inline client/src/shared/header/header.html]: You can simply pass state name to ui-sref:
```html
<a class="wk-logo" ui-sref="home">
```
Please use this simplified version

File: client/src/shared/header/header.controller.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import angular from 'angular';
 const sharedModule = angular.module('shared');
 sharedModule.controller('HeaderController', function HeaderController() {
     this.menuActive = false;
+    this.state = 'home';
```
Comment (gshokanov) [inline client/src/shared/header/header.controller.js]: You don't need this

**Review by gshokanov — CHANGES_REQUESTED** [review]

File: client/src/shared/header/header.controller.js (hunk 2 above)
Comment (Olga1984) [reply client/src/shared/header/header.controller.js]: Updated!

File: client/src/shared/header/header.html (hunk 1 above)
Comment (Olga1984) [reply client/src/shared/header/header.html]: Updated!

**Review by gshokanov — APPROVED** [review]

**Pull request merged**